src/browser/pages/vscode.html

```diff
@@ -48,10 +44,6 @@
 
   <!-- Startup (do not modify order of script tags!) -->
   <script>
-    const parts = window.location.pathname.replace(/^\//g, "").split("/")
```

Comment:
Ah I see how it made lots of things simpler by making the path directly correspond to the file.